python/django_bolt/testing/__init__.py

```
@@ -25,12 +25,15 @@
+from django_bolt.testing.client import AsyncTestClient, BoltTestClientWarning, TestClient
+from django_bolt.testing.dbshare import SharedTestConnectionError
 from django_bolt.testing.websocket import ConnectionClosed, WebSocketTestClient
 
 __all__ = [
     "TestClient",
     "AsyncTestClient",
+    "BoltTestClientWarning",
+    "SharedTestConnectionError",
     "WebSocketTestClient",
     "ConnectionClosed",
 ]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)